src/complexity_measures/missing_dispersion.jl

```diff
@@ -6,31 +6,19 @@ export MissingDispersionPatterns
 
 """
     MissingDispersionPatterns <: ComplexityEstimator
-    MissingDispersionPatterns(est = Dispersion())
+    MissingDispersionPatterns(est = Dispersion()) → mdp
```

Comment:
```suggestion
    MissingDispersionPatterns(o = Dispersion()) → mdp
```

`est` was used when the outcome space was a probabilities estimator, now we use `o` through the code base.